doc/modules/cross_validation.rst

```diff
@@ -468,43 +368,166 @@ validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
 
-Label-Shuffle-Split
```

Comment:
No that would be good. I thought about it and incorrectly assumed they are non-iid in the beginning and added a header for them and later removed it.

I'll add one describing stratification and ping you back.